teuthology/task/internal.py

```diff
@@ -393,6 +393,35 @@ def archive(ctx, config):
                 path = os.path.join(logdir, rem.shortname)
                 misc.pull_directory(rem, archive_dir, path)
 
+                #Check for Coredumps:
```

Comment:
I can go ahead and move it to its own function. Only reason I didn't is the code is called only once